GoogleForCreators/web-stories-wp
# Video: Combine "title" and "assistive text" fields

## Summary

Previously there were two separate fields to set the video's `title` and `alt` attribute, and one pre-publish check for each.

Now there is only one input field for the video description, which sets both `title` and `alt` under the hood.

## Relevant Technical Choices

* Remove the text input for `title`.
* Keep `title` in amp-video tag.
* Use the `alt` update logic to update both `title` and `alt`.
* Updated wording in media edit dialog
* Removed pre-publish check for `title`

## To-do

N/A

## User-facing changes

In the design panel, there is only 1 text field for the video description, which sets both `alt` and `title`.

Also, wording in the media edit dialog is updated

## Testing Instructions

1. Edit video via three dots menu in the media library
1. Change video description in the edit dialog
1. Insert video onto page
1. Verify that the "Accessibility" section in the design panel has only one field that shows that new video description.
1. Preview story on frontend
1. Verify that the video's `alt` and `title` attributes are both set and contain the video description.

After creating the video and updating the video's assistive text, the `alt` attribute should show up under the `<amp-video>` tag, with the same value as `title`.

---

Fixes #4708 
Fixes #5405
